### PR TITLE
[RHICOMPL-977] Add info about what systems are shown

### PR DESCRIPTION
**Pull request by bastilian**
This adds a dismiss-able Info message to the Systems page:

Screenshot 2020-09-28 at 17 34 23

### CODE REVIEW

File: src/SmartComponents/SystemsTable/SystemsTable.js [hunk 1]
```diff
@@ -361,6 +369,13 @@ class SystemsTable extends React.Component {
                 <ErrorPage error={error}/>
             </StateViewPart>
             <StateViewPart stateKey='noError'>
+                { (showComplianceSystemsInfo && showInfoAlert) && <reactCore.Alert
+                    isInline
+                    variant="info"
+                    actionClose={ <reactCore.AlertActionCloseButton onClose={ () => this.dismissInfoAlert() } /> }
```
Comment (akofink) [inline src/SmartComponents/SystemsTable/SystemsTable.js:375]: Is there an ouia ID on this so QE can check that it's on the page or not?

**Review by akofink — APPROVED** [review]
Works as expected, thanks @bastilian! I had one QE-related question.

File: src/SmartComponents/SystemsTable/SystemsTable.js (hunk 1 above)
Comment (bastilian) [reply src/SmartComponents/SystemsTable/SystemsTable.js:375]: The defaults should result in attributes like: `data-ouia-component-type="PF4/Alert" data-ouia-safe="true" data-ouia-component-id="OUIA-Generated-Alert-info-1"`

Comment (Victoremepunto) [reply src/SmartComponents/SystemsTable/SystemsTable.js:375]: wrt OUIA there should be ... 
- `Page` attribute: navigate to a specific page with a unique page type, for example, where the same page with / without a modal open are two different "page types". 
- `Component`: unique identifier (like page) for a specific component.
- `PageSafe` attribute: whether if it's true to interact with the page (the components have loaded)

I think @akofink  comment referred to only "Component" , but all three are very welcome :)

Comment (Victoremepunto) [reply src/SmartComponents/SystemsTable/SystemsTable.js:375]: https://ouia.readthedocs.io/en/latest/README.html#specification-parts

**Pull request merged**